components/dashboard/src/projects/Prebuilds.tsx

```diff
@@ -179,8 +179,8 @@ export default function () {
                         </a>
                     </ItemField>
                     <ItemField className="flex items-center">
-                        <div>
-                            <div className="text-base text-gray-500 dark:text-gray-50 font-medium mb-1">{shortCommitMessage(p.info.changeTitle)}</div>
+                        <div className="truncate">
+                            <div className="text-base text-gray-500 dark:text-gray-50 font-medium mb-1 truncate">{shortCommitMessage(p.info.changeTitle)}</div>
```

Comment:
**thought**: Not sure if I'm missing something here but this works better when using the `truncate` class on both elements. 🤷